src/features/Licenses/LicensesDetailPage/index.jsx

```diff
@@ -0,0 +1,120 @@
+
+  const defaultLicenseInfo = {
+    licenseName: '-',
+    purchasedSeats: '-',
+    numberOfStudents: '-',
+    numberOfPendingStudents: '-',
+  };
+
+  const institution = useSelector((state) => state.main.selectedInstitution);
+  const coursesTable = useSelector((state) => state.courses.table);
+  const stateCourses = useSelector((state) => state.courses);
+  const licenseInfo = useSelector((state) => state.licenses.table.data)
+    .find((license) => license?.licenseId === parseInt(licenseId)) || defaultLicenseInfo;
+
+  const handlePagination = (targetPage) => {
+    setCurrentPage(targetPage);
+    dispatch(updateCurrentPage(targetPage));
+  };
+
+  useEffect(() => {
+    if (institution.id) {
+      dispatch(fetchCoursesData(institution.id, initialPage, stateCourses.filters, licenseId));
+      dispatch(fetchLicensesData(institution.id, initialPage, licenseId));
+    }
+
+    return () => {
+      dispatch(resetCoursesTable());
+    };
+  }, [dispatch, institution.id, licenseId]);
+
+  useEffect(() => {
+    if (institution.id !== undefined && institutionRef.current === undefined) {
+      institutionRef.current = institution.id;
+    }
+
+    if (institution.id !== institutionRef.current) {
+      history.push('/licenses');
+    }
+  }, [institution, history]);
+
+  return (
+    <Container size="xl" className="px-4">
+      <div className="d-flex justify-content-between mb-3 flex-column flex-sm-row">
+        <div className="d-flex align-items-center mb-3">
+          <Link to="/licenses" className="mr-3 link">
+            <i className="fa-solid fa-arrow-left" />
+          </Link>
+          <h3 className="h2 mb-0 course-title">License pool: {licenseInfo.licenseName}</h3>
+        </div>
+
+        <div className="card-container d-flex justify-content-around align-items-center">
+          <div className="d-flex flex-column align-items-center">
+            <p className="title">Purchased</p>
+            <span className="value purchased-seats">
+            {licenseInfo.purchasedSeats}
+            </span>
+          </div>
+          <div className="separator mx-4" />
+            <div className="d-flex flex-column align-items-center">
+              <p className="title">Enrolled</p>
+                <span className="value">
+                  {licenseInfo.numberOfStudents}
+                </span>
+          </div>
+          <div className="separator mx-4" />
+            <div className="d-flex flex-column align-items-center">
+              <p className="title">Remaining</p>
+                <span className="value number-of-pending">
+                  {licenseInfo.numberOfPendingStudents}
+                </span>
+          </div>
+        </div>
+      </div>
+
+      <div className="license-page-content-container">
+        <div className="flex-end">
+          <Button type="button">
+            <a class="btn-license" href="https://www.mindhubpro.com/" target="_blank">Buy a license</a>
```

Comment:
I would suggest to move the link into a constants file

```suggestion
            <a class="btn-license" href={licenseBuyLink} target="_blank">Buy a license</a>
```